src/App.css

```diff
@@ -379,5 +379,60 @@ Footer Styles
     font-weight: bold;
 }
 
+.navigationButton {
+    justify-content: flex-end;
+    background-color: white;
+}
+
+.legendContainer {
+    top: 5vh;
+    left:95%;
+    position: fixed;
+    z-index: 1;
+}
+
+.navContainer {
+    top: 25vh;
```

Comment:
In the comps, this button is directly above the list toggle. Could you move it down to match?